src/agent/onefuzz-agent/src/tasks/coverage/generic.rs

```diff
@@ -322,40 +321,35 @@ impl<'a> TaskContext<'a> {
     }
 }
 
-struct Recorded {
```

Comment:
We're no longer passing around `ModuleCache` ownership, so this type is redundant.